web/styles/user_circles.css

```diff
@@ -41,3 +41,9 @@
         margin: 3.25px auto 0; /* (total height - line height) / 2 = 3.25px */
     }
 }
+
+.user_circle_gray {
+    background-color: var(--grey-550);
+    border-radius: 50%;
+    border: 1px solid var(--grey-550);
+}
```

Comment:
This needs a #design discussion, can you start a discussion for it in `#design`?